app/src/views/Actors.vue

```diff
@@ -2,6 +2,46 @@
   <v-container fluid>
     <BindFavicon />
     <BindTitle value="Actors" />
+    <v-progress-linear
+      :active="pluginLoader"
+      indeterminate
+      absolute
+      top
+      color="deep-purple accent-4"
+    />
+
+    <v-expand-transition>
+      <v-banner app sticky class="mb-2" v-if="selectedActors.length">
+        {{ selectedActors.length }} actors selected
+        <template v-slot:actions>
+          <v-btn v-if="selectedActors.length" text @click="selectedActors = []" class="text-none"
+            >Deselect</v-btn
+          >
+          <v-btn
+            :disabled="selectedActors.length === actors.length"
+            text
+            @click="selectedActors = actors.map(act => act._id)"
+            class="text-none"
+            >Select all</v-btn
+          >
+          <v-btn
+            text
+            @click="runPluginsForSelectedActors"
+            class="text-none"
```

Comment:
Show the button loading, while plugins are being run

```suggestion
            class="text-none"
            :loading="pluginLoader"
```